crm-project/src/main/java/CRM/controller/controllers/SettingController.java

```diff
@@ -18,14 +18,22 @@ public class SettingController {
     @Autowired
     private SettingsFacade settingsFacade;
 
-
-    @GetMapping(value = "user/{userId}/board/{boardId}")
-    public ResponseEntity<Response> getAllUserSettingsInBoard(@DestinationVariable Long userId, @DestinationVariable Long boardId){
-        return null;
+    /**
+     * This method is used to handle HTTP GET requests to the specified URL (settings/get-user-settings-in-board).
+     * The method takes the ids of the user and board and uses them to retrieve all the notifications settings of that user in this specific board.
+     *
+     * @param objectsIdsRequest The ids of the user and the board where the user settings are to be retrieved from.
+     * @return A ResponseEntity object containing the Response object with the retrieved settings and the HTTP status code.
+     */
+    @GetMapping(value = "get-user-settings-in-board")
+    public ResponseEntity<Response> getAllUserSettingsInBoard(@RequestBody ObjectsIdsRequest objectsIdsRequest){
+        Response response = settingsFacade.getAllUserSettingsInBoard(objectsIdsRequest.getUserId(),
+                objectsIdsRequest.getBoardId());
```

Comment:
Pass the objectsIdsRequest instead of all the ids ceparately

Reply:
fixed